### PR TITLE
Best Practice: Keep Controller in Component

### PR DESCRIPTION
**Pull request by ryanrhee**

### CODE REVIEW

**Comment by ryanrhee** [comment]
screenshot 2016-07-14 04 36 17

Attached is a rendering of the proposed page.

File: _docs/42-keep-controller-in-component.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Keep Controller in Component
```
Comment (eczarny) [inline _docs/42-keep-controller-in-component.md]: Maybe _Keep Component and Component Controllers Together_ would be better?

**Comment by eczarny** [comment]
Looks good! Not sure if you can merge so if you can't let me know.

**Comment by ryanrhee** [comment]
> Keep Component and Component Controllers

Will update!

**Comment by ryanrhee** [comment]
@eczarny it's actually too long, jekyll gets confused :| got an alternative name suggestion?

**Comment by ryanrhee** [comment]
Gonna merge as-is, we can rename it later if need be.

File: _docs/42-keep-controller-in-component.md (hunk 1 above)
Comment (adamjernst) [inline _docs/42-keep-controller-in-component.md]: ++

Comment (ryanrhee) [inline _docs/42-keep-controller-in-component.md]: @adamjernst that's too long, it makes jekyll mess up the rendering :/

**Pull request merged**